server/controllers/patients/patchPatientData.js

```diff
@@ -0,0 +1,51 @@
+const {
+  patchPatientDataByIdQuery,
+  patientCheckPhone,
+} = require('../../database/queries');
+const { boomify, patientDataValidation } = require('../../utils');
+
+const patchPatientData = async (req, res, next) => {
+  const { firstName, lastName, phone, email, birthday, diseases } = req.body;
+  const { patientId } = req.params;
+
+  try {
+    const {
+      rows: [patient],
+    } = await patientCheckPhone({ phone });
```

Comment:
Please re-order this query to be after the validation, to check the phone number if it's acceptable and safe and then make the query.